src/boot/efi/pe.c

```diff
@@ -160,6 +166,45 @@ static void locate_sections(
         }
 }
 
+static UINT32 get_compatibility_entry_address(const struct DosFileHeader *dos, const struct PeFileHeader *pe) {
+        UINTN addr = 0, size = 0;
+        static const CHAR8 *sections[] = { (CHAR8 *) ".compat", NULL };
+
```

Comment:
a comment here what this does would be great, as this is totally not obvious for the drive-by reader

Reply:
i mean the function as a whole, not this specific line here